pom.xml

```diff
@@ -403,17 +397,17 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>3.0.1</version>
+                    <version>3.2.1</version>
```

Comment:
latest javadoc version is 3.1.1: https://search.maven.org/artifact/org.apache.maven.plugins/maven-javadoc-plugin (running `mvn javadoc:javadoc` fails)

Reply:
Thanks, I adjusted the version in the wrong line, should be fixed now!